src/file/format/ron.rs

```diff
@@ -0,0 +1,74 @@
+use std::collections::HashMap;
+use std::error::Error;
+
+use ron;
+
+use crate::value::{Value, ValueKind};
+
+pub fn parse(
+    uri: Option<&String>,
+    text: &str,
+) -> Result<HashMap<String, Value>, Box<dyn Error + Send + Sync>> {
+    let value = from_ron_value(uri, &ron::from_str(text)?)?;
+    match value.kind {
+        ValueKind::Table(map) => Ok(map),
+
+        _ => Ok(HashMap::new()),
+    }
+}
+
+fn from_ron_value(
+    uri: Option<&String>,
+    value: &ron::Value,
```

Comment:
Why is `value` passed by reference? From the calling site it looks like we can move it into the function and save a few `clone()` calls here and there, can't we?

Reply:
It was passed by reference only because it was like that in the JSON format. I've modified it to take an owned value.